ffi/capi/src/properties_maps.rs

```diff
@@ -127,10 +127,16 @@ pub mod ffi {
                 self.0.as_borrowed().get_set_for_value(value),
             ))
         }
+        #[diplomat::rust_link(icu::properties::props::GeneralCategory, Enum)]
+        #[diplomat::attr(auto, named_constructor = "general_category")]
+        #[cfg(feature = "compiled_data")]
+        pub fn create_general_category() -> Box<CodePointMapData8> {
```

Comment:
In this file: document compiled data? Probably just make docs for each function that say something like "Construct a general category property map using compiled data"